client/src/App.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState, useEffect, useMemo } from "react";
+import axios from "axios";
+import Login from "./components/Login";
+import Notes from "./components/Notes";
+import { createMuiTheme, ThemeProvider } from "@material-ui/core";
+
+function App() {
+  const [isLogin, setIsLogin] = useState(false);
+
+  const theme = useMemo(() =>
+    createMuiTheme({
+      palette: {
+        type: "light",
+      },
+    })
+  );
+
+  useEffect(() => {
+    const checkLogin = async () => {
+      const token = localStorage.getItem("tokenStore");
+      if (token) {
+        const verified = await axios.get("/users/verify", {
```

Comment:
ideally, the api call should be inside an "api" folder where I export lang nimo siya as function para function call lang basically ag imo buhaton sa frontend na code